src/gsNurbs/gsKnotVector.hpp

```diff
@@ -635,12 +635,14 @@ void gsKnotVector<T>::initUniform( T first,
     for(unsigned i = m_deg - mult_ends + 1, j=0; i!= 0; --i)
```

Comment:
you can also write `for(unsigned i = m_deg - mult_ends + 1, j=0; i!= 0; --i, ++j)`

Reply:
I like it. Incorporated in commit bbe2e008b686ef578b11efda1a53ed4dad7a1012.